desktop/package.json

```diff
@@ -7,10 +7,10 @@
     "start": "npm run build-dev && npm run start-cold",
     "start-hot": "HOT=true NODE_ENV=development node client.js",
     "start-cold": "NODE_ENV=development electron ./dist/main.bundle.js",
-    "build-dev": "NODE_ENV=development NO_SERVER=true DEBUG='express:*' node server.js",
+    "build-dev": "NODE_ENV=production NO_SERVER=true DEBUG='express:*' node server.js",
```

Comment:
we no longer use the NODE_ENV to determine what happens, now its based on development/prod bundles and not the calling context